src/utils.js

```diff
@@ -21,6 +21,10 @@ import {
 } from 'lodash';
 import moment from 'moment';
 
+import {
+  updateTenant,
+  validateUser,
```

Comment:
I think `validateUser` is not currently exported from `stripes-core` so we need a PR there to export it

Reply:
Hi @BogdanDenis here it is
https://github.com/folio-org/stripes-core/pull/1351